content/en/community/squads.md

```diff
@@ -0,0 +1,76 @@
+--- 
+title: "CHT Squads" 
+linkTitle: "Squads"
+weight: 2 
+description: > 
+  How community squads work
+---
+
+## What is a squad?
+
+A CHT Squad is a small, action-oriented team that works together to solve high-priority needs for the community. These teams bring together developers, designers, project managers, and other contributors to build and implement new features and improvements for the CHT.
+
+## How squads work
+
+- Collaborative by design: Anyone can join - whether you are an experienced community contributor or just getting started with the CHT
+- Transparent process: Regular meetings and shared public notes and documentation to keep everyone informed
+- Flexible contributions: Help in whatever way works for you - coding, design, testing, or documentation
+
+## Why join a squad?
+- Work on meaningful projects that make a real difference
+- Learn from and collaborate with other community members
+- Help shape the future of the CHT
+
+
+**The CHT Squad Journey**
+Squads evolve through stages, and anyone can contribute—no matter their experience level. Here are the various stages:
+
+1. Emerging
+An idea sparks! The community identifies a feature or improvement with broad potential.
+
+      **Outputs**:
+      - ✅ Exploratory discussions
+      - ✅ Partners and contributors aligned
+      - ✅ The [Code of Conduct](https://docs.communityhealthtoolkit.org/community/contributing/code-of-conduct/) is shared with all squad members
+
+2. Early Design Discussion
+Feasibility meets vision. Teams explore technical requirements, timelines, and design approaches.
+
+      **Outputs**:
+      - ✅ Wireframes or prototypes
+      - ✅ Resource commitments
+      - ✅ Preliminary scope and timeline
+
+3. Requirements & Design
+Turning ideas into action. Clear specs and mockups are finalized for development.
+
+      **Outputs**:
+      - ✅ Documented requirements
+      - ✅ Approved design doc
+
+4. Development/Building:
+building the solution. 
+Developers implement features based on agreed designs.
+
+      **Output**:
+      - ✅ Working software (GitHub pull requests)
+
+5. Testing
+Ensuring quality. Rigorous manual/automated testing verifies functionality.
+
+      **Outputs**:
+      - ✅ Test reports
+      - ✅ Verified software
+
+6. Release
+Impact in the real world. Features are deployed to real-world users and later merged into core CHT releases.
```

Comment:
```suggestion
The functionality is released and ready to create an impact in the real world. Features can be deployed to users.
```

Reply:
Ensuring that we don't mix up merged, released and deployed, and they are different things.